From cf19b2bd962c7d42c2d5b6d78d7b0015db2b1eb5 Mon Sep 17 00:00:00 2001
From: qianmenx <qianx.meng@intel.com>
Date: Mon, 20 May 2019 16:17:34 +0800
Subject: [PATCH 2318/2328] media: intel-ipu4: ox03a10: change output fps to 30

Message for Open Source:
change output fps to 30

Change-Id: I90ff27597474d773caac34f657170a6ab9eea7ee
Signed-off-by: qianmenx <qianx.meng@intel.com>
(cherry picked from commit 7b47926bba7bf6373447612739af448560de66e5)
Signed-off-by: lijuncax <lijunx.cai@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_common.h | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
index 34dab57..903ad00 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_common.h
@@ -253,8 +253,6 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x3809, CRL_REG_LEN_08BIT, 0x80 },
 	{ 0x380a, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x380b, CRL_REG_LEN_08BIT, 0x40 },
-	{ 0x380e, CRL_REG_LEN_08BIT, 0x05 },
-	{ 0x380f, CRL_REG_LEN_08BIT, 0x40 },
 	{ 0x3810, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x3813, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x381c, CRL_REG_LEN_08BIT, 0x08 },
@@ -522,7 +520,7 @@ static struct crl_register_write_rep ox03a10_1920_1088_12DCG[] = {
 	{ 0x50c2, CRL_REG_LEN_08BIT, 0xb0 },
 	{ 0x5102, CRL_REG_LEN_08BIT, 0xb0 },
 	{ 0x380e, CRL_REG_LEN_08BIT, 0x05 },
-	{ 0x380f, CRL_REG_LEN_08BIT, 0x34 },
+	{ 0x380f, CRL_REG_LEN_08BIT, 0x6d },
 	{ 0x380c, CRL_REG_LEN_08BIT, 0x08 },
 	{ 0x380d, CRL_REG_LEN_08BIT, 0x78 },
 	{ 0x384c, CRL_REG_LEN_08BIT, 0x02 },
-- 
2.7.4

